geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/QueryStrategyDeciderTest.scala

```diff
@@ -33,13 +33,13 @@ import scala.util.Random
 
 //Expand the test - https://geomesa.atlassian.net/browse/GEOMESA-308
 @RunWith(classOf[JUnitRunner])
-class QueryStrategyDeciderTest extends Specification with TestWithDataStore {
+class QueryStrategyDeciderTest extends Specification with TestWithFeatureType {
 
   import org.locationtech.geomesa.filter.ff
 
   override val spec = "nameHighCardinality:String:index=join:cardinality=high,ageJoinIndex:Long:index=join," +
       "heightFullIndex:Float:index=full,dtgJoinIndex:Date:index=join,weightNoIndex:String," +
-      "dtgNoIndex:Date,dtg:Date,*geom:Point:srid=4326"
+      "dtgNoIndex:Date,dtg:Date,*geom:Point:srid=4326;geomesa.index.dtg=dtg"
```

Comment:
Is this additionally bit of indexing necessary?

Reply:
yeah, previously there was an `dtgField: Option[String]` in the base class - i removed that so if you have multiple dates you have to specify the default one in the user data now. i believe this is the only class where that was an issue.